scripts/train.py

```diff
@@ -78,18 +78,21 @@
     out_dir = f"{data_dir}/runs/{args.run_name}/models/"
     mask_names = conf.log_opts.mask_names
 
-    for epoch in range(args.epochs):
+    for epoch in range(1,args.epochs+1):
```

Comment:
Why does epoch need to be 1-indexed?

Reply:
since we were saving on every save_every epoch, using 0-epochs would mean we saved on save_every+1 epoch actually.

Reply:
Ah, that's true, that flag is confusing. I think it'd be clearer to just change the save check -- (epoch + 1) % save_every == 0. After that change is made, I'm happy to see this merged in.

Reply:
Hi Kris, I made these changes, however, there seems to be a merge conflict with the file.